macro/Version.py

```diff
@@ -36,8 +55,23 @@ def execute(macro, args):
                     'New in %s</span>' % version)
 
     def distro_html(distro, distros):
-        active = [distro.encode("iso-8859-1")]
-        inactive = [x.encode("iso-8859-1") for x in distros if not x == distro]
+        preceding_distros = [distro]
+        proceeding_distros = [distro]
+        sorted_distros = sorted(distros)
+        # distro is guaranteed to be in distros
+        assert(distro in distros)
+        distro_index = sorted_distros.index(distro)
+        preceding_distros.extend(sorted_distros[:distro_index])
+        proceeding_distros.extend(sorted_distros[distro_index:][1:])
```

Comment:
Why not remove line 58 and 59 and replace 64 and 65 with the following?

    preceding_distros = sorted_distros[:distro_index + 1]
    proceeding_distros = sorted_distros[distro_index:]